mailpoet/tests/acceptance/Automation/CreateEmailAutomationAndWalkThroughCest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Automation\Engine\Migrations\Migrator;
+use MailPoet\DI\ContainerWrapper;
+use MailPoet\Features\FeatureFlagsController;
+use MailPoet\Features\FeaturesController;
+use MailPoet\Settings\SettingsController;
+
+class CreateEmailAutomationAndWalkThroughCest
+{
+  public function _before() {
+    // @ToDo Remove once MVP is released.
+    $container = ContainerWrapper::getInstance();
+    $migrator = $container->get(Migrator::class);
+    $migrator->createSchema();
+
+    // Set the cron trigger to "WordPress", so we can trigger the cron by visiting the page.
```

Comment:
Just an idea, using `$this->triggerMailPoetActionScheduler()` could probably work too.

Reply:
Thanks. Addressed in [9b286b3](https://github.com/mailpoet/mailpoet/pull/4434/commits/9b286b3d949a2ee01d3b921e776b81ab94fa5220)